### ISSUE TITLE
Pretraining Loss

### ISSUE THREAD

**Issue by yipingji** [fla-org/flash-linear-attention#678]
Hi,

I am pretraining softmax-based LLMs(~200M parameters) using seq_len 2048 and batch size 512. I use Llama2 tokenizer. I notice the pretraining loss starting from ~82 and converges to ~22. I feel like this number is much higher than reported in other papers. Does anyone have clue for this?

Thanks
